src/containers/BooksForm.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+const categories = [
+  'Action',
+  'Biography',
+  'History',
+  'Horror',
+  'Kids',
+  'Learning',
+  'Sci-Fi',
+];
+
+class BooksForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      title: '',
+      category: '',
+    };
+  }
+
+  render() {
+    const { title, category } = this.state;
+    return (
+      <form>
+        <input id="title" value={title} />
+        <select value={category}>
+          {
+            categories.map(cat => (
+              <option key={cat}>
+                {cat}
+              </option>
+            ))
+          }
+        </select>
+      </form>
```

Comment:
- it renders form with a `submit button` see [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone2/Form/RenderForm.png) from pathwright